src/components/dev/DevHud.js

```diff
@@ -0,0 +1,157 @@
+import React, { Component } from 'react';
+import {
+  Menu,
+  Checkbox,
+  Button,
+  Dropdown,
+} from 'semantic-ui-react';
+
+// PROJECT COMPONENTS
+import { HeadingWithDetail } from '../details';
+// Dev components
+import { CustomClient } from '../CustomClient';
+
+// DATA
+import { localizations } from '../../localization/all';
+
+
+const DevMenu = function ({ devProps, funcs, data, state }) {
+
+  var langs    = localizations,
+      langOpts = [];
+
+  for (let key in langs) {
+    var snips = langs[ key ],
+        lang  = {
+          text:  snips.langName,
+          key:   snips.langCode,
+          value: snips.langCode,
+        };
+    langOpts.push(lang);
+  }
+
+  /** @todo If there are enough dev features for it,
+   *    make menu categories collapsible. */
+  return (
+    <div>
+      <Menu.Item header>> Snippets</Menu.Item>
+      <Menu.Item>
+        <Checkbox
+          label    = { `Mark English snippets` }
+          checked  = { devProps.english }
+          onChange = { funcs.english } />
+      </Menu.Item>
+      <Menu.Item>
+        <HeadingWithDetail>
+          <Checkbox
+            label    = { `Mark non-English snippets` }
+            checked  = { devProps.nonEnglish }
+            onChange = { funcs.nonEnglish } />
+          <span>
+            Note: text that doesn't have an underline (for reasons) has no snippets.
```

Comment:
A single quote is not an apostrophe, even if they are sometimes rendered the same. Not a big deal being that this is a developer tool, but elsewhere you should probably use the html code for an apostrophe `&#39;` or `&apos;`.

Reply:
Interesting! Is there a apostrophe on the keyboard somewhere? Without triggering special characters?

Reply:
My gut feeling is it's more robust to keep the source files ASCII and use codes for special characters. At the very least it makes it harder for someone to accidentally use the wrong character / easier to notice errors. I think you can probably copy an apostrophe from the character map, though.

Reply:
@knod There is no apostrophe on the keyboard. Otherwise I agree with what Ethan said.